CryptoTechReminderSystem.Test/Gateway/HarvestGatewayTests.cs

```diff
@@ -108,7 +108,21 @@ public void CanOnlyGetDevelopersWhoAreActive()
                 _harvestApi.Get("/api/v2/users").Responds(jsonWithIsActive);
                 var response = _harvestGateway.RetrieveDevelopers();
                 response.First().FirstName.Should().Be("Dick");
-                response.Count.Should().Be(2);
+                response.Count.Should().Be(6);
+            }
+
+            [Test]
+            public void CanGeDevelopersOnly()
+            {
+                var jsonWithIsActive = File.ReadAllText(
```

Comment:
Could probably rename this to json.

Reply:
removed this test in https://github.com/chubberlisk/crypto_tech/pull/30/commits/997365c3fbec53b1823542213cac8b80db7f17fb